### ISSUE TITLE
Drush installation broken after upgrade

### ISSUE THREAD

**Issue by cmastro-ixiam** [omega8cc/boa#1795]
Drush seems not working after auto-updating and renaming from `BOA-5.2.0-lite` to `BOA-5.2.0-lts`

```
Fri 12 Jul 2024 12:30:09 PM CEST / Devuan.chimaera x86_64 / Aegir lite / Barracuda BOA-5.2.0-lite / Nginx 1.24.0 / PHP-MI 5.6 7.4 8.0 8.1 8.2 / PHP-SE / FPM 8.1 / CLI 8.1 / Percona-5.7.44 localhost
Mon 15 Jul 2024 12:18:43 AM CEST / Devuan.chimaera x86_64 / Aegir lts / Barracuda BOA-5.2.0-lts / Nginx 1.24.0 / PHP-MI 5.6 7.4 8.0 8.1 8.2 / PHP-SE / FPM 8.1 / CLI 8.1 / Percona-5.7.44 localhost
```

Here is the log error:
```
BOA [09:39:32] ==> PROC: sftp_ftps_modern_fix
BOA [09:39:32] ==> PROC: drush_system_install_update
BOA [09:39:32] ==> INFO: Installing supported Drush versions...
BOA [09:39:32] ==> INFO: Drush 8.4.12.9 re-installation required
BOA [09:39:32] ==> PROC: get_drush_eight

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
BOA [09:39:32] ==> INFO: Drush 8.4.12.9 installation complete
BOA [09:39:32] ==> INFO: Drush 10.6.2.9 re-installation required
BOA [09:39:33] ==> PROC: get_drush_ten

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
BOA [09:39:33] ==> INFO: Drush 10.6.2.9 installation complete
BOA [09:39:33] ==> INFO: Drush 11.6.0.9 re-installation required
BOA [09:39:33] ==> PROC: get_drush_eleven

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
BOA [09:39:33] ==> INFO: Drush 11.6.0.9 installation complete
BOA [09:39:33] ==> INFO: Drush 12.1.1.2 re-installation required
BOA [09:39:33] ==> PROC: get_drush_twelve

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
BOA [09:39:33] ==> INFO: Drush 12.1.1.2 installation complete
BOA [09:39:33] ==> PROC: set_drush_perm
chmod: cannot access '/opt/tools/drush/*/drush/drush': No such file or directory
chmod: cannot access '/opt/tools/drush/*/drush/drush.complete.sh': No such file or directory
chmod: cannot access '/opt/tools/drush/*/drush/drush.launcher': No such file or directory
chmod: cannot access '/opt/tools/drush/*/drush/drush.php': No such file or directory
chmod: cannot access '/opt/tools/drush/*/drush/unish.sh': No such file or directory
chmod: cannot access '/opt/tools/drush/*/drush/examples/drush.wrapper': No such file or directory
chmod: cannot access '/opt/tools/drush/*/drush/examples/git-bisect.example.sh': No such file or directory
chmod: cannot access '/opt/tools/drush/*/drush/examples/helloworld.script': No such file or directory
BOA [09:39:33] ==> FAIL: Drush 8.4.12.9 installation failed!
BOA [09:39:33] ==> FAIL: Drush 10.6.2.9 installation failed!
BOA [09:39:33] ==> FAIL: Drush 11.6.0.9 installation failed!
BOA [09:39:33] ==> FAIL: Drush 12.1.1.2 installation failed!
BOA [09:39:33] ==> PROC: aegir_master_install_upgrade

BOA [09:39:33] ==> PROC: if_to_do_fix
```

Is there any solution for this?
Anyone else has happened?

Thanks!

**Comment by omega8cc** [comment]
This looks like connection issue. Have you tried again?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by cmastro-ixiam** [comment]
I have tried several times, but the same problem continues.

I also tried running the curl command that downloads it by replacing the variables:
- `curl -L --max-redirs 10 -k -s --retry 10 --retry-delay 5 -A iCab "http://files.aegir.cc/dev/DEV/drush-8.4.12.8.tar .gz" -o "drush-8.4.12.8.tar.gz"`

And it downloads it without problems:
- `-rw-r--r-- 1 root root 3.0M Jul 15 11:13 drush-8.4.12.8.tar.gz`


To perform the update I'm running `barracuda up-lts system` . This is correct?

**Comment by cmastro-ixiam** [comment]
Performing a deeper analysis, I have found that the variable `_DRUSH_EIGHT_VRN` (for example) is located in the file `BARRACUDA.sh.txt`. But I have searched for this file all over the server and it cannot be found. Is this correct?


In turn, on a test server, I executed the `BOA.sh.txt`:
```
$ wget -q -U iCab http://files.aegir.cc/BOA.sh.txt
$ bash BOA.sh.txt
```
and it created the folder `/opt/boa/` with the file `BARRACUDA.sh.txt`. But when running `barracuda up-lts system` the error is the same.
